packages/communication-react/package.json

```diff
@@ -74,12 +85,14 @@
     "build:cjs": "rollup -c --silent",
     "build:esm": "npm run _if-preprocess && rushx copy-original && rushx preprocess && rushx copy-preprocess  && tspc -project tsconfig.preprocess.json || (if-env COMMUNICATION_REACT_FLAVOR=beta && tspc)",
     "build:watch": "",
+    "build:api": "rushx api-extractor:complete",
     "clean": "rimraf dist && rimraf ../**/preprocessed && rimraf ../**/preprocess-dist",
     "test": "",
     "test:ci-coverage": "",
     "test:coverage": "",
     "check-deps": "node scripts/verifyDependencies",
     "api-extractor": "rushx _by-flavor \"rushx build:esm && rushx _api-extractor:by-flavor\"",
+    "api-extractor:complete": "api-extractor run -c api-extractor-complete.json --local",
```

Comment:
Does this take flavor into consideration?

Reply:
ah this is a good catch will update to account for flavor